toolbox/gui/figure_3d.m

```diff
@@ -1247,12 +1247,13 @@ function SetStandardView(hFig, viewNames)
     % Displaying a surface: Load the SCS field from the MRI
     if isempty(Ranat) && ~isempty(TessInfo) && ~isempty(TessInfo(1).SurfaceFile)
         % Get subject
-        sSubject = db_get('SurfaceFile', TessInfo(1).SurfaceFile);
-        SAnatomy = db_get('AnatomyFile', sSubject.iAnatomy, 'FileName');
+        sAnatomyFile = db_get('AnatomyFile', TessInfo(1).SurfaceFile, 'Subject');
+        sSubject  = db_get('Subject', sAnatomyFile.Subject, 'iAnatomy');
+        [~, sMriFile]  = db_get('AnatomyFile', sSubject.iAnatomy);
         % If there is an MRI associated with it
-        if ~isempty(sSubject) && ~isempty(sSubject.Anatomy) && ~isempty(SAnatomy.FileName)
+        if ~isempty(sSubject) && ~isempty(sMriFile.FileName)

```

Comment:
This will crash if sMriFile is empty (when there is no anatomy file for this subject).

Reply:
Solved in cca37dd55867980ce8937e18b47d101993234186